roles/heat/tasks/main.yml

```diff
@@ -54,6 +54,17 @@
     - heat-engine
   when: os == 'rhel' and openstack_install_method == 'source'
 
+- name: install heat services (rhel source)
```

Comment:
it generated service during rpm installation. However the configuration refers to heat-dist.conf which is added by red hat and contained conflict content with heat.conf. The change is to override the content of service so that heat won't use heat-dist.conf.